docs/source/quickstart.rst

```diff
@@ -167,6 +182,10 @@ To build a Bento, first create a :code:`bentofile.yaml` in your project director
 
 Next, use the :code:`bentoml build` CLI command in the same directory to build a bento.
 
+Please ensure that:
+1. The "service" name is the same as the python file name where you're service is declared
```

Comment:
this is not required by bentoml, but seem like a good idea to recommend users to do this as a contention

Reply:
ok, because this was something weird that wasn't working for me unless the service name was the same as the python file name and it kind of messed me up a little. I must have missed something about how these 2 go together